README.md

```diff
@@ -49,6 +49,9 @@ To create the database run `heroku run python server/init_db.py`
 To sync teams from Zuluru run `heroku run python server/zuluru_sync.py`
 
 
+To backup the database (only the raw games data the rest is calculated by the app) run `python server/backup.py`
```

Comment:
I wonder if we should explicitly note somewhere that we're assuming python3 for all the scripts?

Reply:
I was wondering about that too. For anyone on a ubuntu install (at least for now) the command needs to be `python3` but on windows it is still just `python`. 

I think we should update the setup documentation to specifically mention installing python3 but leave the docs calling it just python. Linux users will need to be savvy enough to know which python is which on their system.